src/api/app/models/workflow_run.rb

```diff
@@ -168,6 +169,10 @@ def push_message
   def tag_push_message
     "Tag #{payload['ref']} got pushed"
   end
+
+  def remove_associated_notifications
```

Comment:
Why don't you setup the association instead of doing this manually?

Reply:
The association with notifications is missing in the workflow run model.

In other "notifiables" we have something like: https://github.com/openSUSE/open-build-service/blob/a96ba2e93bcb26560d12f02cf5558977b688317b/src/api/app/models/bs_request.rb#L82

Reply:
Done